fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -10,6 +10,7 @@ This is the a simple Android application serving the purpose of a shopping list.
 
 <b>Languages:</b>
 * Azerbaijani
+* Bangla
```

Comment:
This list uses the international names of the languages. As such, this should say `Bengali`

Reply:
I like that you already put it in the correct alphabetical order tho.

Reply:
done